wandb/apis/public/runs.py

```diff
@@ -754,10 +754,54 @@ def scan_history(self, keys=None, page_size=1000, min_step=None, max_step=None):
 
     @normalize_exceptions
     def logged_artifacts(self, per_page=100):
+        """Return an iterable collection of all artifacts logged by this run.
+
+        This method retrieves all output artifacts that were logged during the run.
+
+        Arguments:
+            per_page (int, optional): Number of artifacts to fetch per API request. Defaults to 100.
+
+        Returns:
+            RunArtifacts: An iterable collection of all Artifact objects logged as outputs during this run.
+
+        Example:
+            ```python
+            run = api.run("entity/project/run_id")
+            for artifact in run.logged_artifacts():
+                print(f"{artifact.name}:{artifact.version}")
+            ```
+
+        Note:
+            This method returns a paginated result. You can iterate over all artifacts
+            or use list() to collect them into a single list.
+        """
         return public.RunArtifacts(self.client, self, mode="logged", per_page=per_page)
 
     @normalize_exceptions
     def used_artifacts(self, per_page=100):
```

Comment:
if you already here mind adding type annotation

Reply:
Done, added type annotation to the two functions currently in question. Added a ticket to backlog to revisit and check the Run class for any extra type annotations needed.